release_tools/copr_epel7.patch

```diff
@@ -13,11 +13,12 @@
  Requires:	xml-common, openscap-scanner >= 1.2.5
  Obsoletes:	openscap-content < 0:0.9.13
  Provides:	openscap-content
-@@ -41,13 +41,16 @@
+@@ -41,13 +41,17 @@
  
  %prep
  %setup -q
 +mkdir build
++sed -i 's/option(SSG_PRODUCT_OCP4 "If enabled, the OCP4 SCAP content will be built" \${SSG_PRODUCT_DEFAULT})/option(SSG_PRODUCT_OCP4 "If enabled, the OCP4 SCAP content will be built" false)/' CMakeLists.txt
```

Comment:
It should be safer to run the  `cmake` command with `-DSSG_PRODUCT_OCP4:BOOLEAN=OFF`
` cmake -DSSG_PRODUCT_OCP4:BOOLEAN=OFF ..`